easybuild/easyconfigs/d/DIRAC/DIRAC-24.0-foss-2023a.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'CMakeMake'
+
+name = 'DIRAC'
+version = '24.0'
+
+homepage = 'http://www.diracprogram.org'
+description = "DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://zenodo.org/records/10680561/files/']
+sources = ['DIRAC-%(version)s-Source.tgz']
+checksums = ['8943e4a24dbacde3ce3daacc2a6ecebc282702a78fa12638e919b78667b293f5']
+
+builddependencies = [('CMake', '3.26.3')]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('HDF5', '1.14.0')
+]
```

Comment:
`DIRAC-23.0-foss-2023a.eb` also has dependencies on `SciPy-bundle` and `h5py`. Is there a reason they're not included here?